kernel/arch/dreamcast/include/dc/dmac.h

```diff
@@ -0,0 +1,130 @@
+/* KallistiOS ##version##
+
+   dmac.h
+   Copyright (C) 2023 Andy Barajas
+
+*/
+
+/** \file   dc/dmac.h
+    \brief  Macros to access the DMAC(Direct Memory Access Controller) 
```

Comment:
I don't think we really need to spell out what DMA means here...

```suggestion
    \brief  Macros to access the DMA controller
```